packages/manager/apps/pci-block-storage/src/api/hooks/useCatalog.spec.tsx

```diff
@@ -0,0 +1,119 @@
+import { describe, it, vi } from 'vitest';
+import { renderHook } from '@testing-library/react';
+import {
+  TVolumeModel,
+  useAddDescriptionToVolumesCatalog,
+} from '@/api/hooks/useCatalog';
+
+vi.mock('@ovh-ux/manager-pci-common', () => ({
+  useBytes: () => ({ formatBytes: vi.fn() }),
+}));
+
+describe('useAddDescriptionToVolumesCatalog', () => {
+  const baseVolume = {
+    hourlyPrice: {
+      value: 'hour',
+      unit: '€',
+      isLeastPrice: true,
+    },
+    monthlyPrice: {
+      value: 'month',
+      unit: '€',
+      isLeastPrice: true,
+    },
+    iops: 'iops',
+    areIOPSDynamic: true,
+    bandwidth: null,
+    isBandwidthDynamic: true,
+    encrypted: true,
+    capacity: {
+      max: 500,
+    },
+    technicalName: 'tech volume',
+    availabilityZonesCount: 5,
+    showAvailabilityZones: true,
+    name: 'name',
+    displayName: 'displayName',
+    possibleTechnicalNames: ['tech volume'],
+    shouldUseMultiAttachFileSystem: false,
+  } as TVolumeModel;
+
+  it('should map a volume with availibity zone and encryption', () => {
```

Comment:
Please make the test specific to what you are testing, here you are testing a whole object

Reply:
I don't really understand what you mean. How can I be more specific ?